example/.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": [ "axway/env-titanium" ],
+  "parserOptions": {
```

Comment:
This shouldn't be needed here. Just extends from `axway/env-titanium` and use the default options.